source/GameData.cpp

```diff
@@ -1355,16 +1370,18 @@ void GameData::PrintShipTable()
 
 
 
-void GameData::PrintWeaponTable()
+void GameData::PrintGunTable()
 {
 	cout << "name" << '\t' << "cost" << '\t' << "space" << '\t' << "range" << '\t'
-		<< "energy/s" << '\t' << "heat/s" << '\t' << "recoil/s" << '\t'
-		<< "shield/s" << '\t' << "hull/s" << '\t' << "push/s" << '\t'
-		<< "homing" << '\t' << "strength" <<'\n';
+		 << "reload/s" << '\t' << "energy/s" << '\t' << "heat/s" << '\t'
+         << "recoil/s" << '\t' << "shield/s" << '\t' << "hull/s" << '\t'
+         << "push/s" << '\t' << "heat dmg/s" << '\t' << "slow/s" << '\t'
+         << "disruption/s" << '\t' << "ion/s" << '\t' << "homing" << '\t'
+         << "strength" << '\t' << "velocity" << '\n';
```

Comment:
Yeah, I thought about that too but I didn't know if anyone would also think about it so I just guessed and put in the PR, but I can change it.

Reply:
Let's wait for tehhowch.

Reply:
We don't strongly commit to stable stdout emissions. While it is definitely possible to read the ship/weapon table output in a programmatic context, the game doesn't take steps to ensure that the values don't also contain the delimiter, etc.

In the future it could be useful to provide an explicitly machine-parseable output, such as newline-delimited JSON. We could go a step further and also support emitting a schema file with a more verbose data description. It's not a priority of mine at the moment though.